configs/silverstripe.json

```diff
@@ -1,7 +1,8 @@
 {
   "index_name": "silverstripe",
   "start_urls": [
-    "https://docs.silverstripe.org"
+    "https://docs.silverstripe.org",
+    "https://ss-userhelp.netlify.com"
```

Comment:
We would like to stick to one domain only per config. What is the difference between both domain?
You will need to apply for the new URL

Reply:
We have two documentation sites that are running of the same app. The content it displays is determined by an environment variable. If I have to apply for a new domain, that's fine, I guess. It would remove the need for the `context` facet.